### PR TITLE
Use agreed names and CIM values for charge links delivered to frontend

### PR DESCRIPTION
**Pull request by BjarkeMeier**
## References

* #933

### CODE REVIEW

File: source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.WebApi/Files/ChargeLinksMockData.json [hunk 1]
```diff
@@ -3,84 +3,84 @@
     "ChargeType": 3,
     "ChargeId": "40000",
     "ChargeName": "Transmissions nettarif",
-    "ChargeOwnerId": "5790000432752",
-    "ChargeOwnerName": "Energinet (SYO)",
+    "Id": "5790000432752",
+    "Name": "Energinet (SYO)",
```
Comment (HenrikSommer) [inline source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.WebApi/Files/ChargeLinksMockData.json:7]: ChargeOwner
ChargeOwnerName
?

Comment (BjarkeMeier) [reply source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.WebApi/Files/ChargeLinksMockData.json]: Good catch! Thx.

**Comment by LayZeeDK** [comment]
@BjarkeMeier
Can you send a link to/screenshot of how the Swagger UI/OpenAPI spec looks? Specifically, I'm interested in the enum representation.

File: source/Contracts/ChargeLink/ChargeLinkDto.cs [hunk 2]
```diff
@@ -13,31 +13,33 @@
...
+        string ChargeOwner,
         string ChargeOwnerName,
         bool TaxIndicator,
         bool TransparentInvoicing,
-        int Factor,
-        DateTime StartDateTimeUtc,
-        DateTime? EndDateTimeUtc);
+        int ChargeOccurrence,
+        DateTime StartDate,
+        DateTime? EndDate);
```
Comment (LayZeeDK) [inline source/Contracts/ChargeLink/ChargeLinkDto.cs:44]: question: Are we not using `DateTimeOffset`?

Comment (BjarkeMeier) [reply source/Contracts/ChargeLink/ChargeLinkDto.cs:44]: This was based on existing code. I assumed DateTime was decided. Metering point (@thbaan) does the same.
Personally, I'd advocate either DateTimeOffset or NodaTime Instant.
Who can decide?

**Comment by BjarkeMeier** [comment]
> @BjarkeMeier Can you send a link to/screenshot of how the Swagger UI/OpenAPI spec looks? Specifically, I'm interested in the enum representation.

I've added in on the PR description of https://github.com/Energinet-DataHub/greenforce-frontend/pull/197. But for reference it's this one:
image

File: source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.WebApi/Files/ChargeLinksMockData.json [hunk 3]
```diff
@@ -3,84 +3,84 @@
...
     "ChargeName": "Transmissions nettarif",
-    "ChargeOwnerId": "5790000432752",
+    "ChargeOwner": "5790000432752",
     "ChargeOwnerName": "Energinet (SYO)",
     "TaxIndicator": false,
     "TransparentInvoicing": false,
-    "Factor": 1,
-    "StartDateTimeUtc": "2019-12-31T23:00:00Z",
-    "EndDateTimeUtc": null
+    "ChargeOccurrence": 1,
```
Comment (prtandrup) [inline source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.WebApi/Files/ChargeLinksMockData.json:10]: Remember to change this to Quantity

Comment (BjarkeMeier) [reply source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.WebApi/Files/ChargeLinksMockData.json]: Good catch.

File: source/Contracts/ChargeLink/ChargeLinkDto.cs (hunk 2 above)
Comment (BjarkeMeier) [reply source/Contracts/ChargeLink/ChargeLinkDto.cs]: Thx. @jonasdmoeller , @thbaan, and I have decided to go with `DateTimeOffset` as you suggest.

**Review by prtandrup — APPROVED** [review]

**Pull request merged**